### PR TITLE
Provide link to how uptime is managed

### PR DESCRIPTION
**Pull request by simonsabin**
The billing docs don't provide any details about what constitutes uptime. The best I could find is the link in the PR

### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@simonsabin 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/codespaces/getting-started/deep-dive.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ As you develop in your codespace, it will save any changes to your files every f
 
 ### Closing or stopping your codespace
 
-To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command, such as closing the browser tab, running processes will continue until a window of inactivity occurs. 
+To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command, i.e. by closing the browser tab, your browser tab going to sleep, the codespace and its running processes will continue until a window of inactivity occurs, after which the codespace will stop.
```
Comment (timeyoutakeit) [inline content/codespaces/getting-started/deep-dive.md:61]: I think the usual user experience is probably to put their machine to sleep, so I updated the wording to match that and make it more relatable. I also mentioned here what the window of inactivity is. Even though it's mentioned about, this just gives the user some more content.

```suggestion
To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command, i.e. by closing the browser tab or letting your machine go to sleep, the codespace and its running processes will continue until a window of inactivity occurs, after which the codespace will stop.  By default, the window of inactivity is 30 minutes. 
```

**Review by timeyoutakeit — COMMENTED** [review]
Thank you for adding this, this is definitely a very useful change! I made a quick suggestion, let me know what you think!

File: content/codespaces/getting-started/deep-dive.md (hunk 1 above)
Comment (simonsabin) [reply content/codespaces/getting-started/deep-dive.md:61]: Hi Amy, 
Can we simplify this further, in that the browser doesn't need to be closed or machine put in sleep, there is a blanket if there is no activity for 30 minutes your codespace will be stopped. I've noticed that happening when I've been using it. 
Also what happens if using remotely and not in a browser, does the same inactivity window apply.

**Review by pooky5252 — DISMISSED** [review]

File: content/codespaces/getting-started/deep-dive.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ As you develop in your codespace, it will save any changes to your files every f
 
 ### Closing or stopping your codespace
 
-To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command, such as closing the browser tab, running processes will continue until a window of inactivity occurs. 
+To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command, i.e. by closing the browser tab or letting your machine go to sleep, the codespace and its running processes will continue until a window of inactivity occurs, after which the codespace will stop.  By default, the window of inactivity is 30 minutes.
```
Comment (timeyoutakeit) [inline content/codespaces/getting-started/deep-dive.md:61]: @simonsabin Making a suggestion based on your previous comment. What about:

```suggestion
To stop your codespace you can [use the command palette](/codespaces/codespaces-reference/using-the-command-palette-in-codespaces#suspending-or-stopping-a-codespace) (`Shift + Command + P` (Mac) / `Ctrl + Shift + P` (Windows)). If you exit your codespace without running the stop command (for example, closing the browser tab), or if you leave the codespace running without interaction, the codespace and its running processes will continue until a window of inactivity occurs, after which the codespace will stop.  By default, the window of inactivity is 30 minutes. 
```

**Review by timeyoutakeit — DISMISSED** [review]

File: content/codespaces/getting-started/deep-dive.md (hunk 1 above)
Comment (timeyoutakeit) [reply content/codespaces/getting-started/deep-dive.md]: Yep! The inactivity window applies regardless of method of connection to the Codespace.

**Review by ghost — DISMISSED** [review]

**Review by timeyoutakeit — APPROVED** [review]

**Comment by Rizzuto11** [comment]
Sent from Yahoo Mail for iPhone


On Wednesday, September 22, 2021, 1:09 PM, Amy Burns ***@***.***> wrote:




Closed #9538.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Pull request merged**